RevenueCatUI/Templates/V2/ViewHelpers/ShadowModifier.swift

```diff
@@ -42,6 +42,7 @@ struct ShadowModifier: ViewModifier {
     let shape: (any Shape)?
 
     func body(content: Content) -> some View {
+        #if !os(watchOS)
```

Comment:
Executive decision to not support watchOS (right now) in `ShadowModifier` because we don't support watchOS in Paywalls V2 yet anyway